### PR TITLE
Add `Ukrainian` localization

### PR DESCRIPTION
**Pull request by LWJerri**
- [x] index.js
- [x] Translate cheatsheet.json
- [x] Translate general.json
- [x] Translate landing.json
- [x] Translate learn.json
- [x] Translate lessons.json

### CODE REVIEW

**Comment by LWJerri** [comment]
https://github.com/aykutkardas/regexlearn.com/issues/169

File: src/components/LearnFooter/LearnFooter.module.css [hunk 1]
```diff
@@ -102,3 +104,20 @@
...
+
+.FooterCopyrightLink {
+  text-decoration: none;
+  user-select: none;
+  color: var(--txt-2);
+}
+
+.FooterCopyrightLink:hover {
+  opacity: 0.8;
+}
```
Comment (aykutkardas) [inline src/components/LearnFooter/LearnFooter.module.css:123]: Your pull request must not contain any changes to this file.

**Comment by aykutkardas** [comment]
If you fix problems with files that should not change. This pull request seems to be mergeable after a final check. @LWJerri

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "regexlearn",
-  "version": "2.4.4",
+  "version": "2.4.5",
```
[the remark this answers is not in the document]
Comment (LWJerri) [reply package.json]: Solved.

File: src/components/LearnFooter/LearnFooter.module.css (hunk 1 above)
Comment (LWJerri) [reply src/components/LearnFooter/LearnFooter.module.css]: Solved.

File: src/components/LearnFooter/index.tsx [hunk 3]
```diff
@@ -35,6 +35,16 @@ const LearnFooter = ({ steps, step, nextStep, prevStep, success, error }: LearnF
...
+        <div className={styles.FooterCopyright}>
+          <a
+            className={styles.FooterCopyrightLink}
+            target="_blank"
+            rel="noreferrer"
+            href="https://github.com/aykutkardas/regexlearn.com#sponsoring"
+          >
+            <FormattedMessage id="general.becomeSponsor" />
+          </a>
+        </div>
```
[the remark this answers is not in the document]
Comment (LWJerri) [reply src/components/LearnFooter/index.tsx]: Solved.

File: src/components/Social/Social.module.css [hunk 4]
```diff
@@ -1,3 +1,8 @@
+.SocialLinkWrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
```
[the remark this answers is not in the document]
Comment (LWJerri) [reply src/components/Social/Social.module.css]: Solved.

File: src/components/Social/index.tsx [hunk 5]
```diff
@@ -22,7 +22,7 @@ const Social = () => (
     >
       <Icon icon="twitter" size={20} color="white" />
     </a>
-  </>
+  </div>
```
[the remark this answers is not in the document]
Comment (LWJerri) [reply src/components/Social/index.tsx]: Solved.

**Review by aykutkardas — APPROVED** [review]

**Comment by aykutkardas** [comment]
Thanks @LWJerri 🎉

**Pull request merged**